### DISCUSSION TITLE
Can't call reusable workflow with filename only

### DISCUSSION THREAD

**Discussion by charlesrocket** [community/community#50152] [Actions] [answered]
### Select Topic Area

Bug

### Body

Omitting branch/commit breaks the workflow

```
Invalid workflow file: .github/workflows/ci.yml#L100
invalid value workflow reference: no version specified
```

line 100: `uses: user/repo/.github/workflows/cd.yml`

**Comment by airtower-luna** [comment, marked as answer]
If you want call a reusable workflow by file only (at the current commit in the current repository) you need to omit the repository, e.g.:

```yaml
uses: ./.github/workflows/cd.yml
```

If you specify the workflow with the repository, you have to include a ref or commit ID.

See also: https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow

**Reply by charlesrocket** [reply]
Oh this makes sense, thanks!
